pandas/tools/pivot.py

```diff
@@ -24,12 +24,14 @@ def pivot_table(data, values=None, index=None, columns=None, aggfunc='mean',
     ----------
     data : DataFrame
     values : column to aggregate, optional
-    index : a column, Grouper, array which has the same length as data, or list of them.
-        Keys to group by on the pivot table index.
-        If an array is passed, it is being used as the same manner as column values.
-    columns : a column, Grouper, array which has the same length as data, or list of them.
-        Keys to group by on the pivot table column.
-        If an array is passed, it is being used as the same manner as column values.
+    index : a column, Grouper, array which has the same length as data, or list
+        of them.
```

Comment:
this is another numpydoc problem ... 
the type specification should be on one line to have to formatted well in the html docs (as the first line after the colon is formatted in italic), and then second line is the start of the explanation